buildtools/README.md

```diff
@@ -1,11 +1,56 @@
-# Setup
+# AWS.Deploy.Tools.CI
 
-1. Create CF template using `buildtools/ci.template`
-2. Copy output `CodeBuildProjectName` & `OidcRole` output variables.
-3. Create `CI_AWS_ROLE_ARN` repository secret with `OidcRole` value and
-   `CI_AWS_CODE_BUILD_PROJECT_NAME` repository secret with `CodeBuildProjectName`
-   value.
-4. Voila!
+The `AWS.Deploy.Tools.CI` project is used to setup the CI/CD PR check between the [AWS.Deploy.Tools](https://github.com/aws/aws-dotnet-deploy) GitHub Repo and the AWS CodeBuild project that will be executed by the GitHub workflow `AWS CodeBuild CI (codebuild-ci.yml)`.
+
+The workflow will be executed starting from `AWS CodeBuild CI (codebuild-ci.yml)` PR check, which will call a Lambda function in an `AWS Account` (which will act as a load balancer) which will ping 1 or many `Test Runner` accounts to check for availability.
+
+## Main Account (Load Balancer)
+
+We will start by deploying the `Lambda function` to the main testing account. To do this, load up the `AWS.Deploy.Tools.CI` solution in Visual Studio. Make sure the `AWS Visual Studio Toolkit` is installed. Make sure you have AWS credentials with `Admin` access for the main testing account stored in your `credentials` file locally. The Lambda will be deployed using the `AWS Visual Studio Toolkit`.
+* Right-click on the `AWS.Deploy.Tools.CI` project in Visual Studio
```

Comment:
Replace these manual steps with how to deploy with the CLI. Also have a already checked in lambda config file that has all of the requires settings for redeployment. You don't have to manually create the S3 bucket you can use the CLI `--resolve-s3 true` switch to take care of creating the bucket. So there wouldn't have to be anything account specific in the config file.

From this bullet point down to the end could be replaced with something like this.
* dotnet tool update -g Amazon.Lambda.Tools
* dotnet lambda deploy-serverless --region us-west-2 --config-file prod-main-lambda-config.json

Reply:
Done